<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplePlugin</class>
 <widget class="QDockWidget" name="SamplePlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="_getGantry">
        <property name="text">
         <string>Get Gantry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_getUR">
        <property name="text">
         <string>Get UR</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="_calculateTraj">
      <property name="text">
       <string>Calculate Trajectory</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="_executeTraj">
      <property name="text">
       <string>Execute Trajectory</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="_reset">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
